poet_test.go

```diff
@@ -36,7 +38,7 @@ func TestHarness(t *testing.T) {
 	r.NoError(err)
 	cfg.Genesis = time.Now()
 
-	h := newHarness(t, cfg)
+	h := newHarness(t, context.TODO(), cfg)
```

Comment:
"Go question". Wouldn't it be proper to use `context.Background()` instead?

Reply:
`context.TODO()` and `context.Background()` are the same thing. The different names just communicate an intent.

> context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

> context.Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

I was of the opinion that `context.TODO()` would be the appropriate to use in tests, but the documentation says otherwise, then I'll change it.